src/Juvix/Backends/Michelson/Parameterisation.hs

```diff
@@ -161,11 +162,66 @@ checkIntType val (PrimTy (M.Type ty _)) = case ty of
   M.TInt -> True -- TODO bounds?
   _ -> False
 
+primify :: Untyped.T -> PrimTy
+primify t = PrimTy (Untyped.Type t "")
+
 builtinTypes :: P.Builtins PrimTy
-builtinTypes = [] -- FIXME
+builtinTypes =
+  [ (NameSymbol.fromSymbol "Michelson.unit-t", primify Untyped.TUnit),
```

Comment:
this can be done with a `map (\(x, y) -> (NameSymbol.fromSymbol x, primify y))` for reduced duplication

Reply:
Used Arr.*** to do this